lbrynet/lbrynet_daemon/auth/server.py

```diff
@@ -109,20 +122,38 @@ def __init__(self, use_authentication=None):
     def setup(self):
         return NotImplementedError()
 
+    def _set_headers(self, request, data, update_secret=False):
```

Comment:
I should have extended my comment - its useful for the code reviewer if simple housecleaning is in a commit on its own. As it is, its difficult to read through the diff and see whether `_set_headers` was only moved or if it was moved and changed.